### PR TITLE
(PROD-736) Update generic zigbee lock for Yale Key-Free Deadbolt

### PR DESCRIPTION
**Pull request by twack**

### CODE REVIEW

**Comment by workingmonk** [comment]
spacing is all messed up. before the fingerprint line and in between the line. Also, did you test this device? The battery issue was supposed to be fixed. Which means that Yale will now report 200% battery because of the battery hack to fix their first generation devices.

File: devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy [hunk 1]
```diff
@@ -31,7 +31,8 @@
         fingerprint profileId: "0104", inClusters: "0000,0001,0003,0009,000A,0101,0020", outClusters: "000A,0019", manufacturer: "Yale", model: "YRD210 PB DB", deviceJoinName: "Yale Push Button Deadbolt Lock"
         fingerprint profileId: "0104", inClusters: "0000,0001,0003,0009,000A,0101,0020", outClusters: "000A,0019", manufacturer: "Yale", model: "YRD220/240 TSDB", deviceJoinName: "Yale Touch Screen Deadbolt Lock"
         fingerprint profileId: "0104", inClusters: "0000,0001,0003,0009,000A,0101,0020", outClusters: "000A,0019", manufacturer: "Yale", model: "YRL210 PB LL", deviceJoinName: "Yale Push Button Lever Lock"
-    }
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy:34]: Missing the 'Y' in Yale

**Review by tpmanley — CHANGES_REQUESTED** [review]

**Review by tpmanley — APPROVED** [review]

File: devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy (hunk 1 above)
Comment (twack) [reply devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy:34]: fixed

**Comment by workingmonk** [comment]
why didn't you squash the commits?? :'(

**Pull request merged**